internal/clients/http/sources_errors.go

```diff
@@ -8,10 +8,12 @@ import (
 )
 
 var (
-	MoreThanOneAuthenticationForSourceErr = errors.New("more than one authentication")
-	AuthenticationForSourcesNotFoundErr   = fmt.Errorf("authentications for source weren't found in sources app: %w", clients.NotFoundErr)
-	ApplicationNotFoundErr                = fmt.Errorf("application not found is sources app: %w", clients.NotFoundErr)
-	ApplicationTypeNotFoundErr            = fmt.Errorf("application type 'provisioning' not found: %w", clients.NotFoundErr)
-	SourceNotFoundErr                     = fmt.Errorf("source not found: %w", clients.NotFoundErr)
-	AuthenticationSourceAssociationErr    = errors.New("authentication associated to source id not found")
+	SourcesErr                            = errors.New("HTTP client")
+	MoreThanOneAuthenticationForSourceErr = fmt.Errorf("%w: more than one authentication", SourcesErr)
+	ApplicationNotFoundErr                = errors.New(SourcesErr.Error() + fmt.Errorf("application not found is sources app: %w", clients.NotFoundErr).Error())
+	ApplicationTypeNotFoundErr            = errors.New(SourcesErr.Error() + fmt.Errorf("application type 'provisioning' not found: %w", clients.NotFoundErr).Error())
+	SourceNotFoundErr                     = errors.New(SourcesErr.Error() + fmt.Errorf("source not found: %w", clients.NotFoundErr).Error())
+	AuthenticationSourceAssociationErr    = fmt.Errorf("%w: authentication associated to source id not found", SourcesErr)
+	AuthenticationForSourcesNotFoundErr   = errors.New(SourcesErr.Error() + fmt.Errorf("authentications for source weren't "+
+		"found in sources app: %w", clients.NotFoundErr).Error())
```

Comment:
Wait what? :-)

Why don't you use `fmt.Errorf` just like for the client errors above?

Wrapping errors like this via `errors.New` will not work as intended.

Reply:
Unfortunately, fmt.Errorf can not combine two "%w". We would have to wrap it in a different way in that case.

Reply:
Why do we need errors.New?
Why not incorporate SourcesErr.Error() inside the string of fmt.Errorf?

Reply:
When do you need to wrap two errors?  All client/sources/ib errors should only wrap one of the three "root" errors:

```
RootErr = errors.New("a basic error which we can use for errors.Is to find which kind of error it is")
SomeSpecificErr    = fmt.Errorf("%w: xxx", RootErr)
```

And you can of course wrap this again in the code:

    return fmt.Errorf("something is wrong: %w", SomeSpecificErr)

I am not sure what you are trying to achieve here, but concatenating `Error()` calls is not what we want to do, that is giving up on error handling completely.